[0m[[0m[0mdebug[0m] [0m[0m> Exec(testOnly project.KthSmallestTrackerTests, Some(bf252a26-af87-4f04-9b0d-c40f1619ed58), Some(CommandSource(console0)))[0m
[0m[[0m[0mdebug[0m] [0m[0mEvaluating tasks: Test / testOnly[0m
[0m[[0m[0mdebug[0m] [0m[0mRunning task... Cancel: Signal, check cycles: false, forcegc: true[0m
[0m[[0m[0minfo[0m] [0m[0mcompiling 2 Scala sources to /home/huxianfeng/CS557/target/scala-2.13/test-classes ...[0m
[0m[[0m[33mwarn[0m] [0m[0m17 feature warnings; re-run with -feature for details[0m
[0m[[0m[33mwarn[0m] [0m[0mone warning found[0m
[0m[[0m[0minfo[0m] [0m[0m[32mKthSmallestTrackerTests:[0m[0m
[0m[[0m[0minfo[0m] [0m[0m[32mKthSmallestTracker[0m[0m
[0m[[0m[0minfo[0m] [0m[0m[31m- should produce no valid output until finish is asserted and then output the correct kth smallest and addresses *** FAILED ***[0m[0m
[0m[[0m[0minfo[0m] [0m[0m[31m  java.io.IOException: Cannot run program "vcs" (in directory "/home/huxianfeng/CS557/test_run_dir/KthSmallestTracker_should_produce_no_valid_output_until_finish_is_asserted_and_then_output_the_correct_kth_smallest_and_addresses"): error=2, No such file or directory[0m[0m
[0m[[0m[0minfo[0m] [0m[0m[31m  at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1128)[0m[0m
[0m[[0m[0minfo[0m] [0m[0m[31m  at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1071)[0m[0m
[0m[[0m[0minfo[0m] [0m[0m[31m  at os.proc.proc$lzycompute$1(ProcessOps.scala:128)[0m[0m
[0m[[0m[0minfo[0m] [0m[0m[31m  at os.proc.proc$1(ProcessOps.scala:122)[0m[0m
[0m[[0m[0minfo[0m] [0m[0m[31m  at os.proc.spawn(ProcessOps.scala:135)[0m[0m
[0m[[0m[0minfo[0m] [0m[0m[31m  at os.proc.call(ProcessOps.scala:76)[0m[0m
[0m[[0m[0minfo[0m] [0m[0m[31m  at chiseltest.simulator.VcsSimulator$.compileSimulation(VcsSimulator.scala:128)[0m[0m
[0m[[0m[0minfo[0m] [0m[0m[31m  at chiseltest.simulator.VcsSimulator$.createContext(VcsSimulator.scala:95)[0m[0m
[0m[[0m[0minfo[0m] [0m[0m[31m  at chiseltest.internal.BackendExecutive$.start(BackendExecutive.scala:36)[0m[0m
[0m[[0m[0minfo[0m] [0m[0m[31m  at chiseltest.defaults.package$.createDefaultTester(defaults.scala:21)[0m[0m
[0m[[0m[0minfo[0m] [0m[0m[31m  ...[0m[0m
[0m[[0m[0minfo[0m] [0m[0m[31m  Cause: java.io.IOException: error=2, No such file or directory[0m[0m
[0m[[0m[0minfo[0m] [0m[0m[31m  at java.base/java.lang.ProcessImpl.forkAndExec(Native Method)[0m[0m
[0m[[0m[0minfo[0m] [0m[0m[31m  at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:340)[0m[0m
[0m[[0m[0minfo[0m] [0m[0m[31m  at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:271)[0m[0m
[0m[[0m[0minfo[0m] [0m[0m[31m  at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1107)[0m[0m
[0m[[0m[0minfo[0m] [0m[0m[31m  at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1071)[0m[0m
[0m[[0m[0minfo[0m] [0m[0m[31m  at os.proc.proc$lzycompute$1(ProcessOps.scala:128)[0m[0m
[0m[[0m[0minfo[0m] [0m[0m[31m  at os.proc.proc$1(ProcessOps.scala:122)[0m[0m
[0m[[0m[0minfo[0m] [0m[0m[31m  at os.proc.spawn(ProcessOps.scala:135)[0m[0m
[0m[[0m[0minfo[0m] [0m[0m[31m  at os.proc.call(ProcessOps.scala:76)[0m[0m
[0m[[0m[0minfo[0m] [0m[0m[31m  at chiseltest.simulator.VcsSimulator$.compileSimulation(VcsSimulator.scala:128)[0m[0m
[0m[[0m[0minfo[0m] [0m[0m[31m  ...[0m[0m
[0m[[0m[0minfo[0m] [0m[0m[36mRun completed in 3 seconds, 309 milliseconds.[0m[0m
[0m[[0m[0minfo[0m] [0m[0m[36mTotal number of tests run: 1[0m[0m
[0m[[0m[0minfo[0m] [0m[0m[36mSuites: completed 1, aborted 0[0m[0m
[0m[[0m[0minfo[0m] [0m[0m[36mTests: succeeded 0, failed 1, canceled 0, ignored 0, pending 0[0m[0m
[0m[[0m[0minfo[0m] [0m[0m[31m*** 1 TEST FAILED ***[0m[0m
[0m[[0m[31merror[0m] [0m[0mFailed tests:[0m
[0m[[0m[31merror[0m] [0m[0m	project.KthSmallestTrackerTests[0m
[0m[[0m[31merror[0m] [0m[0m(Test / [31mtestOnly[0m) sbt.TestsFailedException: Tests unsuccessful[0m
[0m[[0m[31merror[0m] [0m[0mTotal time: 4 s, completed Apr 10, 2025, 1:48:39 AM[0m
[0m[[0m[0mdebug[0m] [0m[0m> Exec(shell, None, None)[0m
